### ISSUE TITLE
✨ Update label copy of status of cards in "Qualified recruitment processes"

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#7432]
## ✨ Feature

Describe the feature that is being requested

## 🕵️ Details

Add any additional details that could assist with the development of the new feature.

## 🎨 Design File

Include a link to the design file (if it exists).

### Table of statuses
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/619a402b-623d-473a-805b-5f3ee3ed7026)

### Card design
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/ff1a0903-3363-4563-93a4-385216b99007)

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by Jerryescandon** [comment]
Shoutout to @substrae to update the figma file for this issue.

**Comment by petertgiles** [comment]
@Jerryescandon I assume all the status not in this table are unchanged?

**Comment by Jerryescandon** [comment]
Yeah, I think this section only shows cards with these statuses. Other statuses shouldn't appear in this section at all since they are not candidates that have qualified.

**Comment by petertgiles** [comment]
Also, is there a link to the design for these?  For example, the links have different icons and the statuses have different colours.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a609205f-83f5-441e-87fe-7dc3d49f2ae9)
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/80fec8f0-c217-4c23-a81b-70478d0b1ada)

**Comment by substrae** [comment]
@petertgiles @Jerryescandon 

- Figma file for this component is here: https://www.figma.com/file/guHeIIh8dqFVCks310Wv0G/Style-library?type=design&node-id=43-8510&mode=design&t=71vBu8Jzz82jcdzX-4
- To confirm, these changes only apply to recruitments, correct? Not applications.
- Does the label column get applied to the chip element?

**Comment by Jerryescandon** [comment]
Yes, this is only for qualified recruitments.
Yes, I will make it clear on the table.

**Comment by substrae** [comment]
@petertgiles

Putting this on hold until tomorrow so that I can chat with @gobyrne about a few things implementation-wise.

**Comment by substrae** [comment]
@petertgiles 

Sorry! This totally slipped my mind. Here are the latest card designs. I'd like to point out a few decisions that were made in relation to this issue though. These will be summarized an actioned in a separate issue.
- I'm going to be proposing to RATA that we allow users to continue to maintain control over the availability for the "Paused" and "Withdrew" states; this would be mean that setting the user to that state provides them with the control to change their availability back to "Ready to hire"
- The team needs to meet to discuss the scaleable rules that apply to the "Hired" statuses and why they should or should not universally allow candidates to be referred

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/6960386/16f8e043-d28d-4f99-a8c9-dfe8ad2a6a53)
